### ISSUE TITLE
Feedback for WordPress on Azure draft

### ISSUE THREAD

**Issue by snobu** [MicrosoftDocs/architecture-center#4029]
🚨 Looks like you arrived from the internal review site 
🚨 Do not enter Microsoft confidential information here

@vjirovsky, the quality work you put in authoring this article is valuable both for Microsoft and the WordPress community.

Few small things -
* The title [here](https://review.learn.microsoft.com/en-us/azure/architecture/example-scenario/infrastructure/wordpress?branch=pr-en-us-10192#general-wordpress-securityperformance-tips) and throughout the article doesn't use spaces around `&`. It's a little more readable with whitespace around it, i.e. _General WordPress security **&** performance tips_
* [In this section](https://review.learn.microsoft.com/en-us/azure/architecture/example-scenario/infrastructure/wordpress?branch=pr-en-us-10192#general-wordpress-securityperformance-tips) i would rephrase
```
WordPress is a popular target for hackers [...]
```
to
```
Because of its overwhelming popularity, WordPress is a target for hackers [...]
```
Seems more friendly and avoids the risk of sounding adversarial to the project (even if that's not our intention).

* For performance tuning, i would recommend the use of a short list of plugins that we successfully used while testing against the Rook-Ceph backend and gave us excellent visibility into page processing time. The list includes:
    - laps (https://github.com/Rarst/laps) - to see a breakdown of where time is spent serving a page
    - Query Monitor (https://wordpress.org/plugins/query-monitor/) - breakdown of time spent for each SQL query and more (PHP errors, hooks and actions, block editor blocks, enqueued scripts and stylesheets, HTTP API calls)

* Add some database options like MySQL Flexible Server, some folks may not be familiar at all with our managed database offering
* Add more storage options for AKS, like Rook-Ceph (link to both https://rook.io and the AKS-specifc guide https://github.com/Azure/kubernetes-volume-drivers/tree/master/rook-ceph), if customers are willing to manage their own storage nodes. While this increases complexity, someone already using their own RWX storage on AKS may find this familiar

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: e5dbc36d-1197-0b73-d0fe-2c9dd5f1d17d
* Version Independent ID: b2285023-1f19-d037-e1c5-18b0b65009df
* Content: [WordPress on Azure - Azure Architecture Center](https://review.learn.microsoft.com/en-us/azure/architecture/example-scenario/infrastructure/wordpress?branch=pr-en-us-10192&source=docs)
* Content Source: [docs/example-scenario/infrastructure/wordpress.yml](https://github.com/microsoftdocs/architecture-center/blob/main/docs/example-scenario/infrastructure/wordpress.yml)
* Service: **architecture-center**
* Sub-service: **example-scenario**
* GitHub Login: @vjirovsky
* Microsoft Alias: **vjirovsky**

**Comment by ManoharLakkoju-MSFT** [comment]
@snobu 
Thanks for your feedback! We will investigate and update as appropriate.

**Comment by snobu** [comment]
This is internal feedback for @vjirovsky as he progresses from draft to publishing the article. Felt like giving feedback in the way to go here as there's nothing confidential to share.

**Comment by vjirovsky** [comment]
please close this issue
thanks @snobu, your feedback has been included in the latest commit in PR
